presto-main/src/main/java/com/facebook/presto/tracing/NoopTracing.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tracing;
+
+import com.facebook.presto.spi.tracing.Tracer;
+import com.facebook.presto.spi.tracing.Tracing;
+import com.google.inject.Inject;
+
+public class NoopTracing
+        implements Tracing
+{
+    @Inject
+    public NoopTracing() {}
+
+    @Override
+    public Tracer startTracing()
+    {
+        return new NoopTracer();
```

Comment:
instead of creating a new object, declare it as a static const `NOOP_TRACER`and return it here.

Reply:
Name 'NOOP_TRACER' must match pattern '^[a-z][a-zA-Z0-9]*$'.

So I'll just call it noopTracer